core/src/main/scala/cromwell/core/callcaching/docker/DockerClientHelper.scala

```diff
@@ -0,0 +1,27 @@
+package cromwell.core.callcaching.docker
+
+import akka.actor.{Actor, ActorLogging, ActorRef}
+import cromwell.core.actor.RobustClientHelper
+import cromwell.core.callcaching.docker.DockerHashActor.DockerHashResponse
+
+import scala.concurrent.duration.FiniteDuration
+
+trait DockerClientHelper extends RobustClientHelper { this: Actor with ActorLogging =>
+  
+  protected def dockerHashingActor: ActorRef
+  
+  private [core] def dockerResponseReceive: Receive = {
```

Comment:
ToL: I can tell you've turned off your IntelliJ auto-space-formatting for readably formatting akka streams. ;)